spec/facility_spec.rb

```diff
@@ -1,26 +1,142 @@
+      expect(@facility_1.services).to eq([])
+    end
+
+    it "starts with no registered vehicles and no collected fees" do
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@facility_1.collected_fees).to eq(0)
     end
   end
 
   describe '#add service' do
-    it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    it "can add available services" do
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expected = ['New Drivers License', 'Renew Drivers License', 'Vehicle Registration']
+      expect(@facility_1.services).to eq(expected)
+    end
+    it "can't execute services not available" do
+    expect(@facility_2.services).to eq([])
+    expect(@facility_2.register_vehicle(@bolt)).to eq (nil)
+    expect(@facility_2.registered_vehicles).to eq([])
+    expect(@facility_2.collected_fees).to eq(0)
+    expect(@facility_1.administer_written_test(@registrant_1)).to eq(false)
+    expect(@facility_1.administer_road_test(@registrant_2)).to eq(false)
+    expect(@facility_1.renew_drivers_license(@registrant_3)).to eq(false)
+    end
+  end
+
+  describe "#register a vehicle" do
+    it "can register a vehicle" do
+      @facility_1.add_service('Vehicle Registration')
+      expect(@cruz.registration_date).to eq (nil)
+      expect(@facility_1.registered_vehicles).to eq []
+      expect(@facility_1.collected_fees).to eq(0)
+
+      @facility_1.register_vehicle(@cruz)
+
+      expect(@cruz.registration_date).to eq (Date.today)
+      expect(@facility_1.registered_vehicles).to eq([@cruz])
+    end
+
+    it 'can collect corresponding fees' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.collected_fees).to eq(100)
+      @facility_1.register_vehicle(@camaro)
+      @facility_1.register_vehicle(@bolt)
+      expect(@facility_1.collected_fees).to eq(325)
+    end
+  end
+  describe "#issue drivers license and guarantee requirements are met" do
+    it "guarantees requirements to administer written test are met" do
+      expect(@registrant_1.license_data).to eq ({written: false, license: false, renewed: false})
+      expect(@registrant_1.permit?).to eq (true)
```

Comment:
typically we do not leave a space between the eq or the method and the opening parenthesis. I've seen this a lot in your code, so general rule, no space between the opening parenthsis and whatever comes before it (if, elsif, method name, eq, etc)